### PR TITLE
Unit test for java.time 

### PR DESCRIPTION
**Pull request by gaurangkudale**
Added the unit test for package `java.time`  to check

### CODE REVIEW

**Comment by gaurangkudale** [comment]
Hi @cyrille-artho 
I've unit-test for java.time please review it

**Comment by gaurangkudale** [comment]
@cyrille-artho 
Any suggestions?

File: src/tests/java8/java.util.functionTest/ClockTest.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import org.junit.Test;
+import gov.nasa.jpf.util.test.TestJPF;
+import org.junit.Test;
+import java.time.*;
+import java.time.temporal.ChronoUnit;
+import java.util.function.Function;
+import static org.junit.Assert.*;
+import static org.hamcrest.CoreMatchers.*;
+
+
```
Comment (cyrille-artho) [inline src/tests/java8/java.util.functionTest/ClockTest.java:11]: remove duplicate empty lines (only one empty line) here and below

File: src/tests/java8/java.util.functionTest/ClockTest.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+     private Function<String,ZoneId> withZoneId= (timeZoneId -> ZoneId.of(ZoneId.SHORT_IDS.get(timeZoneId)));
+
+
+    @Test
+    public void fixed_clock_test() throws Exception{
+        Instant fixedInstant  = Instant.parse("2014-03-06T21:27:31Z");
+        Clock fixedClock = Clock.fixed(fixedInstant, withZoneId.apply("EST"));
+
+        assertThat(fixedClock.instant(),is(fixedInstant));
+        Thread.sleep(3000l);
```
Comment (cyrille-artho) [inline src/tests/java8/java.util.functionTest/ClockTest.java:24]: Thread.sleep does not work in the same way in JPF as normal; in this case, the test still makes sense, though.

File: src/tests/java8/java.util.functionTest/ClockTest.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    }
+
+    @Test
+    public void  change_clock_duration_test() throws Exception {
+        Clock eastCoastStandardTime = Clock.system(withZoneId.apply("EST"));
+        Clock eastCoastDaylightSavingsTime = Clock.offset(eastCoastStandardTime, Duration.of(1,ChronoUnit.HOURS));
+
+        LocalTime est = LocalTime.now(eastCoastStandardTime);
+        LocalTime edt = LocalTime.now(eastCoastDaylightSavingsTime);
+
```
Comment (cyrille-artho) [inline src/tests/java8/java.util.functionTest/ClockTest.java:49]: the time difference here is rounded down, the result below might be 0L due to the time that elapses between lines 47 and 48?

**Review by cyrille-artho — COMMENTED** [review]
Hi,
I've started a brief review. There are a few things that are not clear to me, as I'm not overly familiar with Java's time API. Do some of the clocks used in the tests advance? If so, we don't know how much time passes between statements.
It may also be problematic to use `Thread.sleep` in tests so if that can be removed, it would be better to remove that.
Finally, please try to make the pull requests with your fork being rebased based on `javapathfinder:master`. The current PR includes over 40 commits from earlier work, which have already been merged.
--
Regards,
Cyrille

**Comment by gaurangkudale** [comment]
Sure I'll do the changes and update you ASAP and I want to work on any javapathfinder project if you have any project available for me please let me know 
I'm really excited and looking forward to do more contribution in JPF and learn new things

**Pull request closed without merging**